### PR TITLE
Email Protection for Windows + Windows compatibility patches

### PR DESCRIPTION
**Pull request by greyivy**
**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/0/1204669886350189/f

## Description
Implements Email Protection for the Windows browser (autofill component). Fixes Windows clickjacking protection and adds "Manage" device API calls.

See also https://github.com/duckduckgo/duckduckgo-autofill/pull/305

## Steps to test
Will be tested as part of https://github.com/duckduckgo/windows-browser/pull/687

### CODE REVIEW

**Review by GioSensation — APPROVED** [review]
I'm approving with a couple of minor notes, but the tests are failing. Do you mind taking a look?

File: src/DeviceInterface/WindowsOverlayDeviceInterface.js [hunk 1]
```diff
@@ -48,6 +58,62 @@ export class WindowsOverlayDeviceInterface extends InterfacePrototype {
...
+                return
+            }
+
+            const activeTooltip = this.uiController?.getActiveTooltip?.()
+            activeTooltip?.focus(event.x, event.y)
+            this.previousScreenX = event.screenX
+            this.previousScreenY = event.screenY
+        })
+
+        return super.addDeviceListeners()
```
[the remark this answers is not in the document]
Comment (greyivy) [reply src/DeviceInterface/WindowsOverlayDeviceInterface.js:88]: I prefer to keep them separate for simplicity's sake. Hopefully it'll make it a bit easier to understand in the future🤞

File: src/DeviceInterface/WindowsOverlayDeviceInterface.js [hunk 2]
```diff
@@ -48,6 +58,62 @@ export class WindowsOverlayDeviceInterface extends InterfacePrototype {
...
+    openManageCreditCards () {
+        return this.deviceApi.notify(new OpenManageCreditCardsCall({}))
+    }
+    /**
+     * @returns {Promise<any>}
+     */
+    openManageIdentities () {
+        return this.deviceApi.notify(new OpenManageIdentitiesCall({}))
+    }
+
```
[the remark this answers is not in the document]
Comment (greyivy) [reply src/DeviceInterface/WindowsOverlayDeviceInterface.js:116]: The MacOS calls currently differ from the Windows ones. I think combining them will make more sense if we migrate the MacOS calls to the new schema system in the future.

[the remark this answers is not in the document]
Comment (GioSensation) [reply src/DeviceInterface/WindowsOverlayDeviceInterface.js:116]: Argh, you're right, different naming. That's unfortunate. Alright, we'll follow up on macOS so they can use the updated names.

**Pull request merged**